packages/wdio-logger/src/node.js

```diff
@@ -119,7 +119,7 @@ export default function getLogger (name) {
     loggers[name].setLevel(logLevel)
     loggers[name].methodFactory = wdioLoggerMethodFactory
     prefix.apply(loggers[name], {
-        template: '%t %l %n:',
+        template: `[${process.env.cid}] %t %l %n:`,
```

Comment:
This would create an empty space for every log where @wdio/logger is not used with the testrunner.